src/test/java/Car1Test.java

```diff
@@ -0,0 +1,55 @@
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class Car1Test {
+    @Test
+    void CarTest() {
+        NumberGenerator2 numberGenerator2 = new NumberGenerator2();
+        Car car1 = new Car("A", 0, numberGenerator2);
+        car1.moving();
+        assertThat(car1.distance).isEqualTo(0);
+
+        NumberGenerator3 numberGenerator3 = new NumberGenerator3();
+        Car car2 = new Car("B", 0, numberGenerator3);
+        car2.moving();
+        assertThat(car2.distance).isEqualTo(1);
+    }
+
+    @Test
+    void RacingTest() {
+        Racing ra = new Racing();
+        ArrayList<String> l = new ArrayList<>();
+        l.add("a");
+        l.add("b");
```

Comment:
제공되는 팩토리메서드를 통해 간결하게 할 수 있겠네요.
```suggestion
        List<String> l = List.of("a", "b");
```